examples/colors/store.rs

```diff
@@ -0,0 +1,142 @@
+use glib::{ParamFlags, ParamSpec, Value};
+use gtk::gdk;
+use gtk::glib;
+use gtk::prelude::*;
+use gtk::subclass::prelude::*;
+
+use crate::color::Color;
+
+mod imp {
+    use std::cell::RefCell;
+
+    use super::*;
+    use gtk::glib::once_cell::sync::Lazy;
+
+    #[derive(Default)]
+    pub struct ColorStore {
+        pub colors: RefCell<Vec<Color>>,
+        pub size: RefCell<u32>,
+    }
+
+    #[glib::object_subclass]
+    impl ObjectSubclass for ColorStore {
+        const NAME: &'static str = "ColorStore";
+        type Type = super::ColorStore;
+        type ParentType = glib::Object;
+        type Interfaces = (gtk::gio::ListModel,);
+    }
+
+    impl ObjectImpl for ColorStore {
+        fn constructed(&self, obj: &Self::Type) {
+            let n_colors = 256 * 256 * 256;
+
+            let mut iter = include_str!("color.names.txt").split('\n');
```

Comment:
This file is huge, I'd prefer to not have that in the repository like this. It's more than 2MB.